allure-web/src/helpers/arrow.js

```diff
@@ -0,0 +1,11 @@
+import {SafeString} from 'handlebars/runtime';
+import {values} from '../util/statuses';
+
+
+export default function(item) {
+    var status = values.slice().reverse().reduce((result, current) => {
+        return item && (item[current] || item.status === current) ? current : result;
+    });
```

Comment:
Do not understand how it works. The given object

```
{
  failed: 2,
  passed: 1
}
```
Will give you `passed` because you are getting whatever comes last, regardless actual value.